AbdoulNuru/Teamwork
# #169023817 Add article endpoint with database

### What does this PR do?

Have the add article endpoint setup

### Description of task to be completed?

Build add article endpoint to help users add articles to the system and write all required tests for that 
endpoint.

### How should this be manually tested?

- Clone this repo to your local repository using `git clone <link>`
- Install all project dependencies using `npm install <package name>`
- In your command line use, `npm start` to run the project
- Now using postman u can send a `post` request to this URL `/api/v2/articles`
- Make sure in your request header you have *key: Content-Type value: application/JSON and 
key: Authorization value: Bearer token*
- And in your request body, you must pass your data in a JSON format.

### What are the relevant pivotal tracker stories?

[#169023817](https://www.pivotaltracker.com/story/show/169023817)

### Screenshots(if appropriate)?
![Screenshot (163)](https://user-images.githubusercontent.com/53453380/66472103-d61c2300-ea8c-11e9-81b2-4b7d00cf9bb6.png)
![Screenshot (164)](https://user-images.githubusercontent.com/53453380/66472107-d6b4b980-ea8c-11e9-8a73-980aff4d4b06.png)
![Screenshot (165)](https://user-images.githubusercontent.com/53453380/66472109-d6b4b980-ea8c-11e9-9a01-a9d88f531fe1.png)
![Screenshot (166)](https://user-images.githubusercontent.com/53453380/66472111-d6b4b980-ea8c-11e9-90e2-85807f5b48cc.png)
![Screenshot (167)](https://user-images.githubusercontent.com/53453380/66478289-51d09c80-ea9a-11e9-97fc-5b56838e86f6.png)
![Screenshot (168)](https://user-images.githubusercontent.com/53453380/66478290-52693300-ea9a-11e9-8b63-c3c2ac6867da.png)
